zcash_client_sqlite/src/wallet.rs

```diff
@@ -3167,11 +3167,9 @@ pub(crate) fn store_decrypted_tx<P: consensus::Parameters>(
     d_tx: DecryptedTransaction<AccountUuid>,
     #[cfg(feature = "transparent-inputs")] gap_limits: &GapLimits,
 ) -> Result<(), SqliteClientError> {
-    // If the transaction is fully shielded, or all transparent inputs are available, set the
-    // fee value.
     let fee = determine_fee(conn, d_tx.tx())?;
```

Comment:
Why did d40b3f31a2837cbc51c8e7766fe749f52daffdce remove this comment? AFAICT it is still accurate.